.github/workflows/close-stale-issues.yml

```
@@ -0,0 +1,28 @@
+name: Close Stale Issues
+
+on:
+  workflow_dispatch:
+
+jobs:
+  close-stale-issues:
+    runs-on: ubuntu-latest
+    permissions:
+      issues: write
+
+    steps:
+      - uses: actions/stale@v10
+        with:
+          days-before-stale: 180 # 6 months
+          days-before-close: 0
+          days-before-pr-stale: -1
+          days-before-pr-close: -1
+          stale-issue-message: ""
+          close-issue-message: >
```

```suggestion
          close-issue-message: |
```

Best not to use a folded (`>`) string block here, we want the newlines to be preserved so use a literal string block (`|`).